apps/librepcb-cli/CMakeLists.txt

```
@@ -0,0 +1,55 @@
+# Enable Qt MOC/UIC/RCC
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTOUIC OFF)
+set(CMAKE_AUTORCC OFF)
+
+# Executable
+add_executable(librepcb_cli
+    # When building on macOS, create a bundle
+    MACOSX_BUNDLE
+
+    # Sources
+    commandlineinterface.cpp
+    main.cpp
+
+    # Headers
+    commandlineinterface.h
+)
+target_include_directories(librepcb_cli PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/../../libs")
+target_link_libraries(librepcb_cli PRIVATE
+    common
+
+    LibrePCB::LibraryManager
+    LibrePCB::ProjectEditor
+)
+set_target_properties(librepcb_cli PROPERTIES OUTPUT_NAME librepcb-cli)
+
+# Install target
+install(TARGETS librepcb_cli
+    BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}
+    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
+)
+
+# macOS bundle configuration
+if(APPLE)
+    # Add icon
+    set(MACOSX_ICON_PATH ${CMAKE_SOURCE_DIR}/img/app/librepcb.icns)
+    target_sources(librepcb_cli PUBLIC ${MACOSX_ICON_PATH})
+    set_source_files_properties(${MACOSX_ICON_PATH} PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
+
+    # Set bundle properties (used for Info.plist)
+    set_target_properties(librepcb_cli PROPERTIES
+        BUNDLE True
+        MACOSX_BUNDLE_GUI_IDENTIFIER org.librepcb.LibrePCB.CLI
```

So far I only saw the recommendation of the format "org.ORGNAME.APPNAME", thus actually I'd not split the application name with a dot. What about this?

```suggestion
        MACOSX_BUNDLE_GUI_IDENTIFIER org.librepcb.LibrePCB-CLI
```

Or is the `.CLI` a common practice?

---

The format is an inverted domain name, so "sub-subdomains" should be fine. However, I don't have any strong opinion and will change it.